### ISSUE TITLE
Would you please increase the maximum length of grouping in Test Explorer?

### ISSUE THREAD

**Issue by PeterHevesi** [microsoft/vstest#5138]
Would it be possible to create another attributes alongside Traits: TraitGroup and SubTraits?
I mean right now, the maximum length of grouping is like 4, if you don't need to use (State, Duration, Target Framework or Environment => these I cannot customize myself, and I am not interested in them), and it is like this, in any order of course: Project > Namespace > Class > Traits.
These 4 attributes are the only one that I can customize myself to group my tests...
And then, it is only very logical for me, that all tests for our website are in 1 project, because it is much easier to manage it.
So basically, the length is 3 for me: Namespace > Class > Traits.
But I would need more groupings indeed. This 2 new attributes would increase the maximum length of grouping by 2 :)
Would this be possible? By adding new attributes, it wouldn't change anything for Traits attribute, right? If someone is still using only Traits, he would not need to change absolutely anything, and everything would work exactly the same, right? 
But other people that needs to have more grouping would be able to use these 2 new attributes :)

**Comment by nohwnd** [comment]
This is a feedback you need to give to Test Explorer team via https://developercommunity.visualstudio.com/home

I believe there is a work item for this already because this is a frequent ask, but I don't know if there is a public feedback ticket for it already.

**Comment by PeterHevesi** [comment]
Ok thank you :)
